sys/log/full/syscfg.yml

```diff
@@ -196,6 +202,10 @@ syscfg.defs:
         restrictions:
             - LOG_FCB_BOOKMARKS
             - LOG_FCB
+    LOG_MAX_TRAILER_LEN:
+        description: >
+            Maximum length of trailer that can be appended to a log entry
+        value: "LF_MAX_ALIGN * 3"
```

Comment:
I'd prefer that this is just a number, not something that is actually an injected code.

Reply:
If the max alignment changes, the max trailer length is going to get affected because alignment plays a big role in fitting the trailer, the implementation of a trailer can be a TLV, a reverse TLV or static data but in order for it to be read successfully from the end, alignment matters. I would prefer to keep it this way if you don't feel strongly.

Reply:
The alignment should be handle by the log framework and I should not really need to care about alignment when setting this number, especially that different log storage can have different alignment requirements.